conftest.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+
+from brainzutils.musicbrainz_db import init_db_engine
+
+
+@pytest.fixture(scope="session")
+def engine():
```

Comment:
This is a pytest _fixture_. It allows you to inject specific data into a test. It's used by adding the name of the fixture as a parameter to a test. It's pretty magic, try not to think about it